conf/application.conf

```diff
@@ -108,5 +108,18 @@ contact-frontend {
   host = "http://localhost:9250"
 }
 
+# Logger
+# ~~~~~
+# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
+
+# Root logger:
+logger.root = ERROR
+
+# Logger used by the framework:
+logger.play = INFO
+
+# Logger provided to your application:
+logger.application = DEBUG
+
```

Comment:
Do we need to add these to application.config ... i assume we just need a application-json-logger.xml & logback.xml ?

Reply:
The microservices I looked in just had the two files, application-json-logger.xml & logback.xml. So yes I'm assuming that's all I need. They didn't add anything else to application.conf